plugins/polio/tasks/vaccine_authorizations_mail_alerts.py

```diff
@@ -136,9 +147,54 @@ def expired_vaccine_authorizations_email_alert():
     return {"vacc_auth_mail_sent_to": mailing_list} if mail_sent else "no_vacc_auth_mail_sent"
 
 
-def vaccine_authorization_update_expired_entries():
+@task_decorator(task_name="expired_vaccine_authorizations_email_alert")
+def send_email_expired_vaccine_authorizations_alert(task=None):
+    past_date = dt.date.today() - timedelta(days=1)
+    vaccine_auths = VaccineAuthorization.objects.filter(expiration_date=past_date)
+    team = get_object_or_404(Team, name="nOPV2 vaccine authorization alerts")
+    mailing_list = [user.email for user in User.objects.filter(pk__in=team.users.all())]
```

Comment:
```suggestion
    mailing_list = team.users.values_list('pk', flat=True)
```

Another way to do the same thing.